package.json

```diff
@@ -8,7 +8,7 @@
     "build": "run-p type-check build-only",
     "build-only": "vite build",
     "build:lib:prod": "npx vite build --mode prod --config vite-dist.config.ts --minify=esbuild --debug && npx babel bundle/lux.dist.mjs --out-file bundle/lux.dist.js",
-    "build:lib:dev": "npx vite build --mode dev --config vite-dist.config.ts --minify=false --base=/dev/main.html/ --debug && cp bundle/lux.dist.mjs bundle/lux.dist.js",
+    "build:lib:dev": "npx vite build --mode staging --config vite-dist.config.ts --minify=false --base=/dev/main.html/ --debug && cp bundle/lux.dist.mjs bundle/lux.dist.js",
```

Comment:
I guess it would make sense to use the `.env.staging` for dev on v4 (`npm start`) also somehow.

Reply:
If we use staging mode for npm start we would use `https://migration.geoportail.lu/getvtstyle` instead of  `https://localhost:8080/getvtstyle` I don't wich one is better.